Basic-Car-Maintenance/Shared/Dashboard/About/AboutAppViewModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  InfoViewModel.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Yashraj jadhav on 01/10/23.
+//
+
+import Foundation
+import SwiftUI
+
+class AboutAppViewModel {
```

Comment:
This entire model isn't needed, it's not used and the pieces that are, I want to remove